install_deps.sh

```diff
@@ -32,16 +32,21 @@ esac
 
 echo ""
 echo "Ensure Git LFS is installed"
+wget https://github.com/git-lfs/git-lfs/releases/download/v3.4.0/git-lfs-linux-amd64-v3.4.0.tar.gz
```

Comment:
doesn't apt install git-lfs work better?

Reply:
People generally do not have permissions to run apt commands on clusters. But not sure why we need this in addition to `git lfs install`. Will `pip install git-lfs` work as well?

Reply:
Good point about lacking `sudo`

You actually do need to install this to run it; `git lfs install` does some preconfiguration stuff but as far as i know doesnt actually download the release